Panic in testing

## Describe the bug
<!-- A clear and concise description of what the bug is. -->
Running tests panics. 

## To Reproduce
<!-- 
Steps to reproduce the behavior:
1. Go to '...'
2. Click on '....'
3. Scroll down to '....'
4. See error 
-->
Checkout main, run a test `make test-long pkgs=./test/email run=TestSender`

## Expected behavior
<!-- A clear and concise description of what you expected to happen. -->
No sys panic

## Screenshots
<!-- If applicable, add screenshots to help explain your problem. -->
```
matt@Carl accounts % make test-long pkgs=./test/email run=TestSender
gofmt -s -l -w ./test/email
git ls-files "*.md" :\!:"vendor/**" | xargs codespell --check-filenames
go vet -composites=false ./test/email
golint ./...
golangci-lint run -c .golangci.yml
go mod tidy
go: downloading github.com/tidwall/pretty v1.0.0
analyze -lockcheck -- ./test/email
go get -d golang.org/x/lint/golint
go: downloading golang.org/x/tools v0.1.5
golint -min_confidence=1.0 -set_exit_status ./test/email
go mod tidy
GORACE='history_size=3 halt_on_error=1 atexit_sleep_ms=2000' go test -race --coverprofile='./cover/cover.out' -v -failfast -tags='testing debug netgo' -timeout=30s ./test/email -run=. -count=1
SIGABRT: abort
PC=0x7ff80bb13112 m=0 sigcode=0

goroutine 0 [idle]:
syscall.rawSyscall(0x10d5c60, 0x0, 0x0, 0x0, 0x0, 0x0, 0x0)
        /usr/local/go/src/runtime/sys_darwin.go:74 +0x4c fp=0xc0001a5138 sp=0xc0001a5110 pc=0x10b280c
syscall.forkAndExecInChild(0xc0002053f0, 0xc00013ee10, 0x3, 0x3, 0xc000184f20, 0x2b, 0x2b, 0x0, 0x0, 0xc0001a5488, ...)
        /usr/local/go/src/syscall/exec_libc2.go:78 +0xeb fp=0xc0001a51f8 sp=0xc0001a5138 pc=0x10c812b
syscall.forkExec(0xc0002053b0, 0xe, 0xc000159520, 0x2, 0x2, 0xc0001a5488, 0x13, 0x9, 0x290)
        /usr/local/go/src/syscall/exec_unix.go:211 +0x487 fp=0xc0001a53a0 sp=0xc0001a51f8 pc=0x10c9587
syscall.StartProcess(...)
        /usr/local/go/src/syscall/exec_unix.go:263
os.startProcess(0xc0002053b0, 0xe, 0xc000159520, 0x2, 0x2, 0xc0001a5700, 0x2a, 0x0, 0x0)
        /usr/local/go/src/os/exec_posix.go:53 +0x44a fp=0xc0001a54e0 sp=0xc0001a53a0 pc=0x110f98a
os.StartProcess(0xc0002053b0, 0xe, 0xc000159520, 0x2, 0x2, 0xc0001a5700, 0x2a, 0x8, 0xc00027c5c8)
        /usr/local/go/src/os/exec.go:106 +0x93 fp=0xc0001a5550 sp=0xc0001a54e0 pc=0x110ee53
os/exec.(*Cmd).Start(0xc000184dc0, 0x1050278, 0x28)
        /usr/local/go/src/os/exec/exec.go:422 +0x8a9 fp=0xc0001a5758 sp=0xc0001a5550 pc=0x1811989
os/exec.(*Cmd).Run(0xc000184dc0, 0x28, 0xc0001a57f0)
        /usr/local/go/src/os/exec/exec.go:338 +0x3d fp=0xc0001a5798 sp=0xc0001a5758 pc=0x181103d
github.com/stripe/stripe-go/v71.getUname(0x1aea4b4, 0x8)
        /Users/matt/Code/go/pkg/mod/github.com/stripe/stripe-go/v71@v71.48.0/stripe.go:1124 +0x178 fp=0xc0001a5810 sp=0xc0001a5798 pc=0x19a6ef8
github.com/stripe/stripe-go/v71.initUserAgent()
        /Users/matt/Code/go/pkg/mod/github.com/stripe/stripe-go/v71@v71.48.0/stripe.go:1148 +0xb9 fp=0xc0001a58e0 sp=0xc0001a5810 pc=0x19a7199
github.com/stripe/stripe-go/v71.init.0()
        /Users/matt/Code/go/pkg/mod/github.com/stripe/stripe-go/v71@v71.48.0/stripe.go:1133 +0x30 fp=0xc0001a58f8 sp=0xc0001a58e0 pc=0x19a70b0
runtime.doInit(0x21dfba0)
        /usr/local/go/src/runtime/proc.go:6265 +0xec fp=0xc0001a5a48 sp=0xc0001a58f8 pc=0x108aa2c
runtime.doInit(0x21e1440)
        /usr/local/go/src/runtime/proc.go:6242 +0x72 fp=0xc0001a5b98 sp=0xc0001a5a48 pc=0x108a9b2
runtime.doInit(0x21df3a0)
        /usr/local/go/src/runtime/proc.go:6242 +0x72 fp=0xc0001a5ce8 sp=0xc0001a5b98 pc=0x108a9b2
runtime.doInit(0x21dbc60)
        /usr/local/go/src/runtime/proc.go:6242 +0x72 fp=0xc0001a5e38 sp=0xc0001a5ce8 pc=0x108a9b2
runtime.doInit(0x21d9860)
        /usr/local/go/src/runtime/proc.go:6242 +0x72 fp=0xc0001a5f88 sp=0xc0001a5e38 pc=0x108a9b2
runtime.main()
        /usr/local/go/src/runtime/proc.go:208 +0x205 fp=0xc0001a5fe0 sp=0xc0001a5f88 pc=0x107ca65
runtime.goexit()
        /usr/local/go/src/runtime/asm_amd64.s:1371 +0x1 fp=0xc0001a5fe8 sp=0xc0001a5fe0 pc=0x10b5921

goroutine 1 [running, locked to thread]:
runtime.asmcgocall(0x10b7da0, 0xc0001a5138)
        /usr/local/go/src/runtime/asm_amd64.s:652 +0x42 fp=0xc0001a50e0 sp=0xc0001a50d8 pc=0x10b5582
runtime.libcCall(0x10b7da0, 0xc0001a5138, 0x1b1aeb8)
        /usr/local/go/src/runtime/sys_libc.go:48 +0x6c fp=0xc0001a5110 sp=0xc0001a50e0 pc=0x109d04c
syscall.rawSyscall(0x10d5c60, 0x0, 0x0, 0x0, 0x0, 0x0, 0x0)
        /usr/local/go/src/runtime/sys_darwin.go:74 +0x4c fp=0xc0001a5138 sp=0xc0001a5110 pc=0x10b280c
syscall.forkAndExecInChild(0xc0002053f0, 0xc00013ee10, 0x3, 0x3, 0xc000184f20, 0x2b, 0x2b, 0x0, 0x0, 0xc0001a5488, ...)
        /usr/local/go/src/syscall/exec_libc2.go:78 +0xeb fp=0xc0001a51f8 sp=0xc0001a5138 pc=0x10c812b
syscall.forkExec(0xc0002053b0, 0xe, 0xc000159520, 0x2, 0x2, 0xc0001a5488, 0x13, 0x9, 0x290)
        /usr/local/go/src/syscall/exec_unix.go:211 +0x487 fp=0xc0001a53a0 sp=0xc0001a51f8 pc=0x10c9587
syscall.StartProcess(...)
        /usr/local/go/src/syscall/exec_unix.go:263
os.startProcess(0xc0002053b0, 0xe, 0xc000159520, 0x2, 0x2, 0xc0001a5700, 0x2a, 0x0, 0x0)
        /usr/local/go/src/os/exec_posix.go:53 +0x44a fp=0xc0001a54e0 sp=0xc0001a53a0 pc=0x110f98a
os.StartProcess(0xc0002053b0, 0xe, 0xc000159520, 0x2, 0x2, 0xc0001a5700, 0x2a, 0x8, 0xc00027c5c8)
        /usr/local/go/src/os/exec.go:106 +0x93 fp=0xc0001a5550 sp=0xc0001a54e0 pc=0x110ee53
os/exec.(*Cmd).Start(0xc000184dc0, 0x1050278, 0x28)
        /usr/local/go/src/os/exec/exec.go:422 +0x8a9 fp=0xc0001a5758 sp=0xc0001a5550 pc=0x1811989
os/exec.(*Cmd).Run(0xc000184dc0, 0x28, 0xc0001a57f0)
        /usr/local/go/src/os/exec/exec.go:338 +0x3d fp=0xc0001a5798 sp=0xc0001a5758 pc=0x181103d
github.com/stripe/stripe-go/v71.getUname(0x1aea4b4, 0x8)
        /Users/matt/Code/go/pkg/mod/github.com/stripe/stripe-go/v71@v71.48.0/stripe.go:1124 +0x178 fp=0xc0001a5810 sp=0xc0001a5798 pc=0x19a6ef8
github.com/stripe/stripe-go/v71.initUserAgent()
        /Users/matt/Code/go/pkg/mod/github.com/stripe/stripe-go/v71@v71.48.0/stripe.go:1148 +0xb9 fp=0xc0001a58e0 sp=0xc0001a5810 pc=0x19a7199
github.com/stripe/stripe-go/v71.init.0()
        /Users/matt/Code/go/pkg/mod/github.com/stripe/stripe-go/v71@v71.48.0/stripe.go:1133 +0x30 fp=0xc0001a58f8 sp=0xc0001a58e0 pc=0x19a70b0
runtime.doInit(0x21dfba0)
        /usr/local/go/src/runtime/proc.go:6265 +0xec fp=0xc0001a5a48 sp=0xc0001a58f8 pc=0x108aa2c
runtime.doInit(0x21e1440)
        /usr/local/go/src/runtime/proc.go:6242 +0x72 fp=0xc0001a5b98 sp=0xc0001a5a48 pc=0x108a9b2
runtime.doInit(0x21df3a0)
        /usr/local/go/src/runtime/proc.go:6242 +0x72 fp=0xc0001a5ce8 sp=0xc0001a5b98 pc=0x108a9b2
runtime.doInit(0x21dbc60)
        /usr/local/go/src/runtime/proc.go:6242 +0x72 fp=0xc0001a5e38 sp=0xc0001a5ce8 pc=0x108a9b2
runtime.doInit(0x21d9860)
        /usr/local/go/src/runtime/proc.go:6242 +0x72 fp=0xc0001a5f88 sp=0xc0001a5e38 pc=0x108a9b2
runtime.main()
        /usr/local/go/src/runtime/proc.go:208 +0x205 fp=0xc0001a5fe0 sp=0xc0001a5f88 pc=0x107ca65
runtime.goexit()
        /usr/local/go/src/runtime/asm_amd64.s:1371 +0x1 fp=0xc0001a5fe8 sp=0xc0001a5fe0 pc=0x10b5921

rax    0x0
rbx    0x4c1b600
rcx    0x7ff7bfefeef8
rdx    0x0
rdi    0x303
rsi    0x6
rbp    0x7ff7bfefef20
rsp    0x7ff7bfefeef8
r8     0x0
r9     0x0
r10    0x0
r11    0x246
r12    0x303
r13    0x50
r14    0x6
r15    0x16
rip    0x7ff80bb13112
rflags 0x246
cs     0x7
fs     0x0
gs     0x0
=== RUN   TestSender
coverage: [no statements]
panic: test timed out after 30s

goroutine 347 [running]:
testing.(*M).startAlarm.func1()
        /usr/local/go/src/testing/testing.go:1700 +0x11f
created by time.goFunc
        /usr/local/go/src/time/sleep.go:180 +0x52

goroutine 1 [chan receive]:
testing.(*T).Run(0xc000001200, 0x1aebd5c, 0xa, 0x1b19d40, 0x0)
        /usr/local/go/src/testing/testing.go:1239 +0x610
testing.runTests.func1(0xc000001200)
        /usr/local/go/src/testing/testing.go:1511 +0xa7
testing.tRunner(0xc000001200, 0xc0001a5c78)
        /usr/local/go/src/testing/testing.go:1193 +0x203
testing.runTests(0xc00000e0f0, 0x22b4b20, 0x2, 0x2, 0xc07081bdbb465cb8, 0x6fe060263, 0x22d9300, 0xc000130148)
        /usr/local/go/src/testing/testing.go:1509 +0x613
testing.(*M).Run(0xc000020100, 0x0)
        /usr/local/go/src/testing/testing.go:1417 +0x3b4
main.main()
        _testmain.go:87 +0x357

goroutine 3 [select]:
go.mongodb.org/mongo-driver/x/mongo/driver/topology.(*Topology).selectServerFromSubscription(0xc000332000, 0x1e66958, 0xc00003a640, 0xc0003022a0, 0x1e5ce60, 0xc000074580, 0xc000302240, 0x0, 0x0, 0x1e5ce60, ...)
        /Users/matt/Code/go/pkg/mod/go.mongodb.org/mongo-driver@v1.8.1/x/mongo/driver/topology/topology.go:446 +0x2e7
go.mongodb.org/mongo-driver/x/mongo/driver/topology.(*Topology).SelectServer(0xc000332000, 0x1e66958, 0xc00003a640, 0x1e5ce60, 0xc000074580, 0x0, 0x0, 0x0, 0x0)
        /Users/matt/Code/go/pkg/mod/go.mongodb.org/mongo-driver@v1.8.1/x/mongo/driver/topology/topology.go:391 +0x17e
go.mongodb.org/mongo-driver/x/mongo/driver.Operation.selectServer(0xc00006a750, 0x1aebd5c, 0xa, 0x1e603a8, 0xc000332000, 0xc00006a760, 0x1e5ce60, 0xc000074580, 0x0, 0x0, ...)
        /Users/matt/Code/go/pkg/mod/go.mongodb.org/mongo-driver@v1.8.1/x/mongo/driver/operation.go:236 +0xee
go.mongodb.org/mongo-driver/x/mongo/driver.Operation.getServerAndConnection(0xc00006a750, 0x1aebd5c, 0xa, 0x1e603a8, 0xc000332000, 0xc00006a760, 0x1e5ce60, 0xc000074580, 0x0, 0x0, ...)
        /Users/matt/Code/go/pkg/mod/go.mongodb.org/mongo-driver@v1.8.1/x/mongo/driver/operation.go:241 +0x98
go.mongodb.org/mongo-driver/x/mongo/driver.Operation.Execute(0xc00006a750, 0x1aebd5c, 0xa, 0x1e603a8, 0xc000332000, 0xc00006a760, 0x1e5ce60, 0xc000074580, 0x0, 0x0, ...)
        /Users/matt/Code/go/pkg/mod/go.mongodb.org/mongo-driver@v1.8.1/x/mongo/driver/operation.go:308 +0x118
go.mongodb.org/mongo-driver/x/mongo/driver/operation.(*CreateIndexes).Execute(0xc000306d00, 0x1e66958, 0xc00003a640, 0xc00000e468, 0x0)
        /Users/matt/Code/go/pkg/mod/go.mongodb.org/mongo-driver@v1.8.1/x/mongo/driver/operation/createIndexes.go:115 +0x36f
go.mongodb.org/mongo-driver/mongo.IndexView.CreateMany(0xc0003021e0, 0x1e66958, 0xc00003a640, 0xc00000e1e0, 0x1, 0x1, 0x0, 0x0, 0x0, 0x0, ...)
        /Users/matt/Code/go/pkg/mod/go.mongodb.org/mongo-driver@v1.8.1/mongo/index_view.go:273 +0xed1
github.com/SkynetLabs/skynet-accounts/database.ensureDBSchema(0x1e66958, 0xc00003a640, 0xc000302120, 0xc0003161c0, 0x0, 0x0)
        /Users/matt/Code/SkynetLabs/accounts/database/database.go:292 +0x30ff
github.com/SkynetLabs/skynet-accounts/database.NewCustomDB(0x1e66958, 0xc00003a640, 0x1aebd5c, 0xa, 0x1ae6e51, 0x5, 0x1af0796, 0x10, 0x1aeb44c, 0x9, ...)
        /Users/matt/Code/SkynetLabs/accounts/database/database.go:131 +0x1f9
github.com/SkynetLabs/skynet-accounts/test/email.TestSender(0xc000001380)
        /Users/matt/Code/SkynetLabs/accounts/test/email/sender_test.go:25 +0x225
testing.tRunner(0xc000001380, 0x1b19d40)
        /usr/local/go/src/testing/testing.go:1193 +0x203
created by testing.(*T).Run
        /usr/local/go/src/testing/testing.go:1238 +0x5d8

goroutine 4 [select]:
go.mongodb.org/mongo-driver/x/mongo/driver/topology.(*rttMonitor).start(0xc000020180)
        /Users/matt/Code/go/pkg/mod/go.mongodb.org/mongo-driver@v1.8.1/x/mongo/driver/topology/rtt_monitor.go:95 +0x286
created by go.mongodb.org/mongo-driver/x/mongo/driver/topology.(*rttMonitor).connect
        /Users/matt/Code/go/pkg/mod/go.mongodb.org/mongo-driver@v1.8.1/x/mongo/driver/topology/rtt_monitor.go:66 +0x6a

goroutine 5 [select]:
go.mongodb.org/mongo-driver/x/mongo/driver/topology.(*Server).update.func3()
        /Users/matt/Code/go/pkg/mod/go.mongodb.org/mongo-driver@v1.8.1/x/mongo/driver/topology/server.go:501 +0x205
go.mongodb.org/mongo-driver/x/mongo/driver/topology.(*Server).update(0xc000334000)
        /Users/matt/Code/go/pkg/mod/go.mongodb.org/mongo-driver@v1.8.1/x/mongo/driver/topology/server.go:555 +0x5ea
created by go.mongodb.org/mongo-driver/x/mongo/driver/topology.(*Server).Connect
        /Users/matt/Code/go/pkg/mod/go.mongodb.org/mongo-driver@v1.8.1/x/mongo/driver/topology/server.go:214 +0x34d

goroutine 6 [sync.Cond.Wait]:
sync.runtime_notifyListWait(0xc00003a910, 0xc000000000)
        /usr/local/go/src/runtime/sema.go:513 +0xf8
sync.(*Cond).Wait(0xc00003a900)
        /usr/local/go/src/sync/cond.go:56 +0x90
go.mongodb.org/mongo-driver/x/mongo/driver/topology.(*pool).createConnections.func2(0x0, 0x0, 0x0)
        /Users/matt/Code/go/pkg/mod/go.mongodb.org/mongo-driver@v1.8.1/x/mongo/driver/topology/pool.go:610 +0x19d
go.mongodb.org/mongo-driver/x/mongo/driver/topology.(*pool).createConnections(0xc000338000, 0x1e66958, 0xc00003a940, 0xc000028450)
        /Users/matt/Code/go/pkg/mod/go.mongodb.org/mongo-driver@v1.8.1/x/mongo/driver/topology/pool.go:637 +0x162
created by go.mongodb.org/mongo-driver/x/mongo/driver/topology.(*pool).connect
        /Users/matt/Code/go/pkg/mod/go.mongodb.org/mongo-driver@v1.8.1/x/mongo/driver/topology/pool.go:158 +0x22a

goroutine 7 [sync.Cond.Wait]:
sync.runtime_notifyListWait(0xc00003a910, 0xc000000001)
        /usr/local/go/src/runtime/sema.go:513 +0xf8
sync.(*Cond).Wait(0xc00003a900)
        /usr/local/go/src/sync/cond.go:56 +0x90
go.mongodb.org/mongo-driver/x/mongo/driver/topology.(*pool).createConnections.func2(0x0, 0x0, 0x0)
        /Users/matt/Code/go/pkg/mod/go.mongodb.org/mongo-driver@v1.8.1/x/mongo/driver/topology/pool.go:610 +0x19d
go.mongodb.org/mongo-driver/x/mongo/driver/topology.(*pool).createConnections(0xc000338000, 0x1e66958, 0xc00003a940, 0xc000028450)
        /Users/matt/Code/go/pkg/mod/go.mongodb.org/mongo-driver@v1.8.1/x/mongo/driver/topology/pool.go:637 +0x162
created by go.mongodb.org/mongo-driver/x/mongo/driver/topology.(*pool).connect
        /Users/matt/Code/go/pkg/mod/go.mongodb.org/mongo-driver@v1.8.1/x/mongo/driver/topology/pool.go:158 +0x22a

goroutine 8 [select]:
go.mongodb.org/mongo-driver/x/mongo/driver/topology.(*pool).maintain(0xc000338000, 0x1e66958, 0xc00003a940, 0xc000028450)
        /Users/matt/Code/go/pkg/mod/go.mongodb.org/mongo-driver@v1.8.1/x/mongo/driver/topology/pool.go:744 +0x54a
created by go.mongodb.org/mongo-driver/x/mongo/driver/topology.(*pool).connect
        /Users/matt/Code/go/pkg/mod/go.mongodb.org/mongo-driver@v1.8.1/x/mongo/driver/topology/pool.go:161 +0x2ec
FAIL    github.com/SkynetLabs/skynet-accounts/test/email        30.311s
FAIL
make: *** [test-long] Error 1
```

## Version Information (please complete the following information):
 <!-- 
 - OS: [e.g. iOS]
 - Browser [e.g. chrome, safari]
 - Git Commit or Git Tag [e.g. 22] 
-->
git commit 43bda9145b95ea4cc3a86d8d167ca2735fae18f1
Running on MacOS in VSCode

## Additional context
<!-- Add any other context about the problem here. -->

--- a/Makefile
+++ b/Makefile
@@ -19,7 +19,7 @@ clean:
 	@docker stop genenv || true && docker rm --force genenv
 ifneq ("$(OS)","Windows_NT")
 # Linux
-	rm -rf cover output 
+	rm -rf cover output
 else
 # Windows
 	- DEL /F /Q cover output
@@ -144,11 +144,11 @@ test:
 
 test-long: lint lint-ci
 	@mkdir -p cover
-	GORACE='$(racevars)' go test -race --coverprofile='./cover/cover.out' -v -failfast -tags='testing debug netgo' -timeout=30s $(pkgs) -run=. -count=$(count)
+	GORACE='$(racevars)' go test -race --coverprofile='./cover/cover.out' -v -failfast -tags='testing debug netgo' -timeout=60s $(pkgs) -run=$(run) -count=$(count)
 
 # These env var values are for testing only. They can be freely changed.
 test-int: test-long start-mongo
-	GORACE='$(racevars)' go test -race -v -tags='testing debug netgo' -timeout=300s $(integration-pkgs) -run=. -count=$(count)
+	GORACE='$(racevars)' go test -race -v -tags='testing debug netgo' -timeout=600s $(integration-pkgs) -run=$(run) -count=$(count)
 	-make stop-mongo
 
 # test-single allows us to run a single integration test.
@@ -157,7 +157,7 @@ test-int: test-long start-mongo
 test-single: export COOKIE_HASH_KEY="7eb32cfab5014d14394648dae1cf4e606727eee2267f6a50213cd842e61c5bce"
 test-single: export COOKIE_ENC_KEY="65d31d12b80fc57df16d84c02a9bb62e2bc3b633388b05e49ef8abfdf0d35cf3"
 test-single:
-	GORACE='$(racevars)' go test -race -v -tags='testing debug netgo' -timeout=300s $(integration-pkgs) -run=$(RUN) -count=$(count)
+	GORACE='$(racevars)' go test -race -v -tags='testing debug netgo' -timeout=300s $(integration-pkgs) -run=$(run) -count=$(count)
 
 # docker-generate is a docker command for env var generation
 #

